scripts/azureml-assets/setup.py

```diff
@@ -18,6 +18,8 @@
       "marshmallow>=3.19",
       "tenacity>=8.2.2",
       "azure-ai-ml>=1.9.0",
+      "azure-storage-blob>=12.18",
+      "azure-identity>=1.14",

```

Comment:
We should already have some versions of these being installed as dependencies of azure-ai-ml, so only include them here if we need to override versions (I hope not!).